low-level/sspl-ll.spec

```diff
@@ -97,6 +98,8 @@ SSPL_DIR=/opt/seagate/%{product_family}/sspl
     ln -sf $SSPL_DIR/low-level/manifest_support_bundle /usr/bin/manifest_support_bundle
     ln -sf $SSPL_DIR/low-level/framework $SSPL_DIR/low-level/solution/lr2/
     ln -sf $SSPL_DIR/low-level/solution $SSPL_DIR/extension/solution
+    ln -sf $SSPL_DIR/low-level/sspl_support_bundle $SSPL_DIR/bin/sspl_support_bundle
+    ln -sf $SSPL_DIR/low-level/sspl_support_bundle /usr/bin/sspl_support_bundle
```

Comment:
avoid to create sspl_support bundle soft link in /usr/bin/ as @cdeshmukh comment. (https://github.com/Seagate/cortx-monitor/pull/493/files#r686046387).
@cdeshmukh Can we remove the manifest support bundle soft link also from here?
As we are moving manifest bundle code in utils? If Yes.
@jjxsg Can you please remove line no. 98 and from line no 133?

Reply:
Removed soft link in /usr/bin/
For manifest @cdeshmukh please confirm.

Reply:
Done